src/ggrc/assets/javascripts/models/mappers/mappings-ggrc.js

```diff
@@ -496,6 +496,16 @@
         });
       }, 'CustomAttributeDefinition')
     },
+    // used by the Custom Roles admin panel tab
+    Roleable: {
+      custom_role_definitions: CustomFilter(
+        'WHAT_HERE???',  // TODO: or a different mapper helper? which?
```

Comment:
@Smotko Blocked here, what "loader helper" (if that's a right term) should I use here? Did not find anything suitable, and not even `CustomFilter` worked, as it relies on another mapping to filter on it.

I basically just want to fetch `AccessControlRole` instances that belong to a chosen model type. Is there a good way to do it, or do I have to wait for the updates to Search API?

The model itself seems to work,  `AccessControlRole.findAll()` succeeds (with an empty list, though, b/c no data in my DB yet).